AmazonChimeSDK/AmazonChimeSDK/audiovideo/AudioVideoConfiguration.swift

```diff
@@ -15,12 +15,15 @@ import Foundation
     public static let defaultCallKitEnabled: Bool = false
     public static let defaultEnableAudioRedundancy: Bool = true
     public static let defaultVideoMaxResolution: VideoResolution = .videoResolutionHD
+    public static let defaultReconnectTimeoutMs: Int = 180 * 1000
```

Comment:
Why not 120ms like JS SDK?

Reply:
180 is the current chime SDK default value, if use 120, it will be a behaviour change.